plugins/events/classes/event_processor.php

```diff
@@ -62,6 +62,20 @@ public static function process_event(\core\event\base $event) {
                     'grade' => $eventdata['other']['finalgrade']
                 ];
                 break;
+            case '\\local_fe_events_moodle\\event\\fe_event':
```

Comment:
I'm curious: If we were to have this processor in the same plugin that generates the event, would things work? (we don't have to modify the PR to reflect that implementation, I just want to see if there's by chance any problem with a single plugin both generating and consuming an event).